### PR TITLE
Added first airgun Content Host test

### PR DESCRIPTION
**Pull request by abalakh**
Depends on SatelliteQE/airgun#177
```python
 pytest -v tests/foreman/ui_airgun/test_contenthost.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 1 item
2018-08-14 13:49:02 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_contenthost.py::test_positive_end_to_end PASSED [100%]

========================== 1 passed in 464.23 seconds ==========================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_contenthost.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            session.organization.select(org.name)
+            # Ensure content host is searchable
+            assert session.contenthost.search(
+                vm.hostname)[0]['Name'] == vm.hostname
+            chost = session.contenthost.read(vm.hostname)
+            # Ensure all content host fields/tabs have appropriate values
+            assert chost['details']['name'] == vm.hostname
+            assert (
+                chost['details']['content_view'] ==
+                repos_collection._setup_content_data['content_view']['name']
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_contenthost.py]: access to private attribute, I have created a property in this PR https://github.com/SatelliteQE/robottelo/pull/6218/files#diff-002cc267b83726c3b9cc934babf4a2a3R587
not merged yet

File: tests/foreman/ui_airgun/test_contenthost.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            )
+            assert chost['details']['lce'][lce.name][lce.name]
+            assert (
+                chost['details']['registered_by'] == 'Activation Key {}'
+                .format(repos_collection._setup_content_data[
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_contenthost.py]: same as above

File: tests/foreman/ui_airgun/test_contenthost.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            assert actual_repos == expected_repos
+            # Install package
+            result = session.contenthost.execute_package_action(
+                vm.hostname,
+                'Package Install',
+                FAKE_0_CUSTOM_PACKAGE_NAME,
+            )
+            assert result['result'] == 'success'
+            # Ensure package installed
+            package = session.contenthost.search_package(
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_contenthost.py]: packages

File: tests/foreman/ui_airgun/test_contenthost.py (hunk 1 above)
Comment (ldjebran) [reply tests/foreman/ui_airgun/test_contenthost.py]: you can you create that property , and one of us should rebase :)

**Comment by abalakh** [comment]
Addressed.
```python
pytest -v tests/foreman/ui_airgun/test_contenthost.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 1 item                                                              2018-10-08 17:45:22 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item

tests/foreman/ui_airgun/test_contenthost.py::test_positive_end_to_end PASSED [100%]

========================== 1 passed in 728.75 seconds ==========================
```

**Pull request merged**